evap/staff/views.py

```diff
@@ -1449,36 +1479,11 @@ def evaluation_person_management(request, evaluation_id):
                 if not importer_log.has_errors():
                     save_import_file(excel_file, request.user.id, import_type)
 
-        else:
-            if "replace" in operation:
-                deleted_person_count, deletion_message = helper_delete_users_from_evaluation(evaluation, operation)
-
-            if "import" in operation:
-                file_content = get_import_file_content_or_raise(request.user.id, import_type)
-                importer_log = import_persons_from_file(
-                    import_type, evaluation, test_run=False, file_content=file_content
-                )
-                delete_import_file(request.user.id, import_type)
-            elif "copy" in operation:
-                copy_form.evaluation_selection_required = True
-                if copy_form.is_valid():
-                    import_evaluation = copy_form.cleaned_data["evaluation"]
-                    importer_log = import_persons_from_evaluation(
-                        import_type, evaluation, test_run=False, source_evaluation=import_evaluation
-                    )
-
-            if "replace" in operation:
-                importer_log.add_success(
-                    format_html(deletion_message, deleted_person_count, evaluation.full_name),
-                    category=ImporterLogEntry.Category.RESULT,
-                )
-
-            importer_log.forward_messages_to_django(request)
+        elif import_or_copy_participants(request, operation, import_type, evaluation, copy_form):
```

Comment:
I would prefer if we would not have the call to `import_or_copy_participants` in the condition here - what do you think about making this an `else` and assigning `successfully_processed = import_or_copy_participants(...)` in there?